commands/lint.js

```diff
@@ -0,0 +1,18 @@
+import chalk from 'chalk';
+import cp from 'child_process';
+
+export const lint = (lintSources = 'src/') => {
+  const path = __dirname + '/../node_modules/eslint/bin/eslint.js';
+
+  const eslint = cp.spawn(path, [lintSources], {
```

Comment:
Can you add some inline above line 7? cp.spawn isn't  very clear to those who haven't used it, makes it more accessible to new contributors.